data/bash-completion/fwupdmgr

```diff
@@ -6,6 +6,7 @@ _fwupdmgr_cmd_list=(
 	'device-test'
 	'device-emulate'
 	'device-wait'
+	'devicelist-upload'
```

Comment:
@superm1 I'm not 100% happy with this name, better ideas welcome

Reply:
Have you already thought about making this opt in through GNOME specifically?   Like gnome initial setup configures it?
I think this is going to be really interesting data, but buried in fwupd as a command I don't think it's going to get a lot of use.

Reply:
(But nonetheless I do think you want to have a command either way)

Here's some other ideas:

`send-devicelist`
`report-devices`
`report-system`

`vote-system`
`vote-devices`
`vote-devicelist`

And actually this begs a bigger question - why do we need to have so many different kinds of reports?  Security report, device update success/fail, device list?   It's a lot.

As a user I feel like we should ask them if they want to report data to help make fwupd better and make it binary.  Send them all at once.

Reply:
> Have you already thought about making this opt in through GNOME specifically

I'm not sure my fireproof suit is up for that.

> but buried in fwupd as a command I don't think it's going to get a lot of use

Fair. What about linking it into `fwupdmgr get-devices` -- that's at least 1% of users.

Reply:
> I'm not sure my fireproof suit is up for that.

I mean yeah it's going to require a lot of conversation.. But there's so much value in steering future conversations!

> What about linking it into fwupdmgr get-devices -- that's at least 1% of users.

Yeah that ties into my point of we should just have one prompt for all the things.

Reply:
How about a series of prompts (need word smithing, but you get the jist):

1. `Would you like to upload data about your system to make fwupd better?  Y/N` and default to Y.
3. If N `Do you want to be asked again?`
2. If Y `How much data do you want to share?  [1-3]` and default to 3
3. Do you want to be asked each time?

For the explanations:
"3" could be everything
"2" could be security reports and device pass fail
"1" could be device pass fail
"0" could be nothing

Reply:
Let's think about the fwupd-refresh action a bit more; I think that's the direction we want to go here. I think it makes sense to merge this PR first, and get some early data from a few hundred people -- then opt-in harder when we know how to pre-process the data client side to reduce server bandwidth and load.